<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDevoluciones</class>
 <widget class="QMainWindow" name="frmDevoluciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Devoluciones</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabdetails">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/res/document-edit.png</normaloff>:/icons/res/document-edit.png</iconset>
       </attribute>
       <attribute name="title">
        <string notr="true"/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>No Documento</string>
            </property>
            <property name="buddy">
             <cstring>txtDocumentNumber</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtDocumentNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Cliente</string>
            </property>
            <property name="buddy">
             <cstring>txtBill</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page_5">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QComboBox" name="cbClient"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_6">
             <widget class="QLineEdit" name="txtClient">
              <property name="geometry">
               <rect>
                <x>4</x>
                <y>4</y>
                <width>105</width>
                <height>25</height>
               </rect>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Factura</string>
            </property>
            <property name="buddy">
             <cstring>txtClient</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtBill">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Fecha</string>
            </property>
            <property name="buddy">
             <cstring>dtPicker</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dtPicker">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tabledetails">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Observaciones</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="txtObservations">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Totales</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Subtotal: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Impuesto: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lblTaxes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>C$ 0.00</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Costo Total: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="lblCost">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>C$ 0.00</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_13">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Total Devolución: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="lblTotal">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>C$ 0.00</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lblSubtotal">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>C$ 0.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabnavigation">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/res/table.png</normaloff>:/icons/res/table.png</iconset>
       </attribute>
       <attribute name="title">
        <string notr="true"/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tablenavigation">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Buscar</string>
            </property>
            <property name="buddy">
             <cstring>txtSearch</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSearch"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionCancel"/>
   <addaction name="actionPreview"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionGoFirst"/>
   <addaction name="actionGoPrevious"/>
   <addaction name="actionGoNext"/>
   <addaction name="actionGoLast"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-new.png</normaloff>:/icons/res/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-preview.png</normaloff>:/icons/res/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Previsualizar</string>
   </property>
  </action>
  <action name="actionGoFirst">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/go-first.png</normaloff>:/icons/res/go-first.png</iconset>
   </property>
   <property name="text">
    <string>Ir al Primer Registro</string>
   </property>
  </action>
  <action name="actionGoPrevious">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/go-previous.png</normaloff>:/icons/res/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Ir al Registro Anterior</string>
   </property>
  </action>
  <action name="actionGoNext">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/go-next.png</normaloff>:/icons/res/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Ir al siguiente registro</string>
   </property>
  </action>
  <action name="actionGoLast">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/go-last.png</normaloff>:/icons/res/go-last.png</iconset>
   </property>
   <property name="text">
    <string>Ir al ultimo registro</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-save.png</normaloff>:/icons/res/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/dialog-cancel.png</normaloff>:/icons/res/dialog-cancel.png</iconset>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-edit.png</normaloff>:/icons/res/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/edit-delete.png</normaloff>:/icons/res/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/edit-copy.png</normaloff>:/icons/res/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/edit-cut.png</normaloff>:/icons/res/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/edit-paste.png</normaloff>:/icons/res/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>txtDocumentNumber</tabstop>
  <tabstop>dtPicker</tabstop>
  <tabstop>tabledetails</tabstop>
  <tabstop>txtObservations</tabstop>
  <tabstop>txtBill</tabstop>
  <tabstop>txtClient</tabstop>
  <tabstop>cbClient</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tablenavigation</tabstop>
  <tabstop>txtSearch</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
